python/sglang/srt/entrypoints/openai/serving_chat.py

```diff
@@ -826,8 +841,47 @@ def _process_tool_calls(
         tools: List[Any],
         tool_call_parser: Optional[str],
         finish_reason: Dict[str, Any],
+        tool_choice: Optional[Union[str, Any]] = None,
     ) -> tuple[Optional[List[ToolCall]], str, Dict[str, Any]]:
         """Process tool calls in the response"""
+        if tool_choice == "required":
+            try:
+                tool_call_data = json.loads(text)
+                tool_calls = []
+                for i, call_info in enumerate(tool_call_data):
+                    tool_id = f"call_{uuid.uuid4().hex[:24]}"
+                    tool_calls.append(
+                        ToolCall(
+                            id=tool_id,
+                            index=i,
+                            function=FunctionResponse(
+                                name=call_info['name'], 
+                                arguments=json.dumps(call_info['parameters'], ensure_ascii=False)
+                            ),
+                        )
+                    )
+                return tool_calls, "", finish_reason
+            except Exception as e:
```

Comment:
it seems the only exception here is Json error? Please specific the exceptions here.